lib/tasks/schedule.rake

```diff
@@ -130,7 +130,7 @@ task({ :reminder_rsvp=> :environment}) do
     events3.each do |event|
         if event.group.rsvp_send_before == "1"
             
-            event.rsvps.where(:accepted => false).each do |rsvp|
+            event.rsvps.where(:accepted => false).each do |rsvp| 
```

Comment:
With sending reminders, it's usually nice to have a column that let's you know if the thing has been sent yet. That way you can be extra sure you won't spam your users. For RSVPs there might be multiple reminders being sent, but there is still usually a set total number of reminders they should get.